mftf/2.3/reporting.md

```diff
@@ -0,0 +1,368 @@
+---
```

Comment:
Does this need frontmatter for group, mftf-release: 2.3.8, etc? Just checking.

Reply:
the front matter is defined in the [_config.yml](https://github.com/magento/devdocs/blob/master/_config.yml#L107-L112) 
it is a part of Multi-Repo implementation